sdk/test-utils/recorder/test/node/utils.spec.ts

```diff
@@ -713,4 +718,92 @@ describe("NodeJS utils", () => {
       });
     });
   });
+
+  describe("applyRequestBodyTransformations", () => {
```

Comment:
Can you add tests with multiple requests? I want us to ensure that this works on:

- Multiple matches.
- Files with one match and multiple other network requests that shouldn't be matched.

Reply:
This method is called with a single recording(request+response) only. "multiple other network requests" are being covered.
The tests below cover the following
- applyRequestBodyTransformations with default transforms - should get affected
- applyRequestBodyTransformations with default transforms - recording unchanged for no client-request-id
- applyRequestBodyTransformations with default transforms - recording unchanged for json request bodies